interface/main/finder/dynamic_finder.php

```diff
@@ -8,17 +8,22 @@
 // of the License, or (at your option) any later version.
 
 require_once("../../globals.php");
+require_once "$srcdir/user.inc";
+require_once "$srcdir/MedEx/API.php";
```

Comment:
Is this Medex API being used anywhere?

Reply:
I didn't use it.

Reply:
lets remove it then; likely a copy/paste error